lib/zoho_sign/response_handler.rb

```diff
@@ -3,47 +3,56 @@
 module ZohoSign
   # Internal class to centralize the code related to the response of the Zoho Sign API
   class ResponseHandler
-    def initialize(params)
-      @params = params || {}
+    def initialize(response, http_response = nil)
+      @response = response || {}
+      @http_response = http_response
     end
 
     def body
-      @params
+      @response
     end
 
     def data(key)
-      @params[key]
+      return unless @response.is_a?(Hash)
+
+      @response[key]
     end
 
     def success?
-      @params[:status] == "success"
+      return false unless @response.is_a?(Hash)
+
+      @response[:status] == "success"
```

Comment:
```suggestion
      @http_response == 200
```

Reply:
the issue is that I can't manage to pass the `http_reponse` as a params all the time when calling `ResponseHandler.new`
https://github.com/wecasa/zoho_sign/blob/handle-html-response/lib/zoho_sign/base_record.rb#L69
here, I don't have it in the body, so maybe I need to find a way to add it here too (but I didn't find it easily)